godocgen.go

```diff
@@ -160,28 +143,70 @@ func start() error {
 		return fmt.Errorf("could not copy to %v: %v", staticDir, err)
 	}
 
-	packages, err := list(c)
+	packages, err := list(*c)
 	if err != nil {
 		return fmt.Errorf("cannot list packages: %v", err)
 	}
 
 	for _, pkg := range packages {
-		err := writePackage(c, packages, pkg)
+		err := writePackage(*c, packages, pkg)
 		if err != nil {
 			return fmt.Errorf("could not write package %v: %v", pkg.Name, err)
 		}
 	}
 
-	if err := makeIndexes(c); err != nil {
+	if err := makeIndexes(*c); err != nil {
 		return fmt.Errorf("could not generate index.html files: %v", err)
 	}
-	if err := makeHome(c, packages); err != nil {
+	if err := makeHome(*c, packages); err != nil {
 		return fmt.Errorf("could not generate index.html files: %v", err)
 	}
 
 	return nil
 }
 
+func parseConfig(opts options) (*Config, error) {
```

Comment:
Is there a specific reason you're returning a pointer here? Especially since you're using `*c` a lot later on.

Reply:
I guess that I started some change and finally I propagated it everywhere 😛 Thanks for pointing it out, I'll do the change